colonist/processor/src/test/java/com/joom/colonist/processor/integration/ErrorReporterExtensions.kt

```diff
@@ -0,0 +1,15 @@
+package com.joom.colonist.processor.integration
+
+import com.joom.colonist.processor.ErrorReporter
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+
+fun ErrorReporter.shouldContainErrorMessage(message: String) {
+  getErrors().shouldHaveSize(1)
+  getErrors().first().message shouldBe message
+}
```

Comment:
```suggestion
fun ErrorReporter.shouldContainSingleErrorMessage(message: String) {
  with(getErrors()) {
    shouldHaveSize(1)
    single().message shouldBe message
  }
}
```
Minor, just add more clarification and strictness